include/openssl/ocsp.h

```diff
@@ -59,13 +59,30 @@ DECLARE_ASN1_FUNCTIONS(OCSP_REQUEST)
 DECLARE_ASN1_FUNCTIONS(OCSP_SIGNATURE)
 DECLARE_ASN1_FUNCTIONS(OCSP_REQINFO)
 
+// Blocking OCSP request handler: now a special case of non-blocking I/O.
+// Combines OCSP_sendreq_new() with as many calls of OCSP_sendreq_nbio() as
+// needed and then OCSP_REQ_CTX_free(), with a response header maximum line
+// length 4k. It waits indefinitely on a response.
+//
+// WARNING: This does not support setting a timeout or adding your own HTTP
+// headers and is retained only for compatibility.
+// Use OCSP_sendreq_nbio and manually set your own timeout instead. You can also
```

Comment:
How do you set a timeout for OCSP_sendreq_nbio?

Reply:
I actually meant setting a timer in the consumer’s own code, since these APIs don’t support it. I’ll have to clarify that:sweat_smile:

Reply:
Can we add a test that this works as expected?